language/diem-framework/modules/DiemConsensusConfig.move

```diff
@@ -27,8 +26,6 @@ module DiemFramework::DiemConsensusConfig {
 
     /// Allows Diem root to update the config.
     public fun set(dr_account: &signer, config: vector<u8>) {
-        DiemTimestamp::assert_operating();
```

Comment:
Silly question: I'm guessing this is so that we can set the config on genesis itself, and not only after genesis has occurred?

Reply:
exactly